src/formio.common.ts

```diff
@@ -31,6 +31,36 @@ export interface FormioForm {
   submissionAccess?: AccessSetting[];
 }
 
+export interface GridColumn {
+  label: string;
+  path: string;
+  renderCell?(cellValue: any, component?: any): string;
+}
+
+export interface ComponentInstance {
+  component: ExtendedComponentSchema;
+  getView: any;
+  id: string;
+  type: string;
+  asString?(value: any): string;
+}
+
+export interface GridHeader {
+  component: ComponentInstance | null;
+  key: string;
+  label: string;
+  renderCell?(cellValue: any, component?: any): string;
+}
+
+export interface FormRow {
```

Comment:
@vlad-shusterman could you also please make sure that you use this `FormRow` type everywhere where row / rows are used in angular-formio?